Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkTransformationManager.java

```diff
@@ -56,8 +58,48 @@ public ITransformedNetwork getTransformedNetwork(String networkId) throws Quadri
             return null;
         }
 
-        networkJSon = transformer.transformNetwork(networkTopNodesList); 
+        networkJSon = transformer.transformNetwork(networkTopNodesList);
 
         return networkJSon;
     }
+
+	@Override
+	public ITransformedNetwork getTransformedNetworkOfProject(String projectId)
+			throws QuadrigaStorageException {
+
+		List<INetwork> networkList;
+		ITransformedNetwork networkJSon;
+
+		List<INetworkNodeInfo> networkNodeInfoList = new ArrayList<INetworkNodeInfo>();
+
+		try {
+			networkList = networkManager.getNetworksInProject(projectId);
+		} catch (QuadrigaStorageException e) {
+			logger.error("DB Error while getting networks a project id", e);
+			return null;
+		}
+
+		if (networkList == null) {
+			return null;
+		}
+
+		for (INetwork network: networkList) {
+
+			try {
+				List<INetworkNodeInfo> localNetworkNodeInfoList =
+						networkManager.getNetworkTopNodes(network.getNetworkId());
+				if (localNetworkNodeInfoList != null) {
+					networkNodeInfoList.addAll(localNetworkNodeInfoList);
+				}
+			} catch (QuadrigaStorageException e) {
+				logger.error("DB Error while getting network top nodes for a network with id - " +
```

Comment:
don't swallow the exception. The user doesn't know what happening if you just log the exception and forget about it.